R/updateNullModelApproxScoreSE.R

```diff
@@ -0,0 +1,246 @@
+## estimate the ratio (score.se.ratio) of the true SE(score) to the fast SE(score)
+## update the null model to contain this parameter
+## this needs to be run before using the approx.score.se option in assocTestSingle
+## this is based off of the SAIGE variance approximation
+
```

Comment:
I don't love the name of this function because it's so long. Very open to other suggestions.